sql/resource_sql_endpoint_test.go

```diff
@@ -80,19 +81,68 @@ func TestResourceSQLEndpointCreate(t *testing.T) {
 	assert.Equal(t, "d7c9d05c-7496-4c69-b089-48823edad40c", d.Get("data_source_id"))
 }
 
+func TestResourceClassicSQLEndpointCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/sql/warehouses",
+				ExpectedRequest: SQLEndpoint{
```

Comment:
can you rewrite the test to work with `map[string]any`, this will show if the fix is exactly there and we're properly serializing the request payload.